MapboxSearch/offline/src/androidTest/java/com/mapbox/search/offline/OfflineSearchEngineIntegrationTest.kt

```diff
@@ -564,9 +563,14 @@ internal class OfflineSearchEngineIntegrationTest {
                 center = serverResult.center,
                 distanceMeters = serverResult.distanceMeters
             )
-            return fixedResult == serverResult &&
-                    expected.center.approximatelyEquals(serverResult.center) &&
-                    compareDistanceMeters(expected.distanceMeters, serverResult.distanceMeters)
+
+            if ((fixedResult == serverResult) &&
```

Comment:
Given that we have unstable data for now, It will be better to compare search results by fields, not the whole objects, so that you'll be able quickly see in test reports which field was wrong. Also, this will allow you easily comment out fields comparison that we know won't pass because of a bug we're seeing now.

```
        fun assertEquals(result1: OfflineSearchResult, result2: OfflineSearchResult) {
            assertEquals(result1.name, result2.name)
            assertEquals(result1.descriptionText, result2.descriptionText)
            assertTrue(result1.coordinate.approximatelyEquals(result2.coordinate))
            // compare other fields

            assertEquals(result1.address, result2.address)
        }

        fun assertEquals(address1: OfflineSearchAddress?, address2: OfflineSearchAddress?) {
            if (address1 == null || address2 == null) {
                assertNull(address1)
                assertNull(address2)
                return
            }
            assertEquals(address1.country, address2.country)
            // compare other fields
        }
```

What do you think?

Reply:
I've done this. Please take a look.